### PR TITLE
Fixes #9825 - foreman can connect to remote SMTP

### PR DESCRIPTION
**Pull request by lzap**
I enabled this by default.

### CODE REVIEW

File: foreman.te [hunk 1]
```diff
@@ -283,6 +290,10 @@ optional_policy(`
     mta_send_mail(passenger_t)
 ')
 
+tunable_policy(`passenger_can_connect_smtp',`
+    corenet_tcp_connect_smtp_port(passenger_t)
```
Comment (stbenjam) [inline foreman.te:294]: Does this cover 465 and 587 too?

Comment (domcleal) [inline foreman.te:294]: It appears so.

<pre>
# semanage port -l | grep smtp_port_t
smtp_port_t                    tcp      25, 465, 587
</pre>

Comment (lzap) [inline foreman.te:294]: Yup, sorry for the delay.

**Comment by domcleal** [comment]
Merged as 2112b96, thanks @lzap.

**Pull request closed without merging**